app/src/components/blocks/feedbackForm/FeedbackForm.tsx

```diff
@@ -0,0 +1,135 @@
+
+        sendEmail(formData)
+            .then(() => {
+                setStage(SUBMISSION_STAGE.Successful);
+                setResult(formData);
+            })
+            .catch((e) => {
+                setStage(SUBMISSION_STAGE.Failure);
+            });
+    };
+
+    const feedbackContentProps = renameRefKey(
+        register(FORM_FIELDS.feedbackContent, {
+            required: 'Please enter your feedback',
+        }),
+        'textareaRef',
+    );
+    const howSatisfiedProps = renameRefKey(
+        register(FORM_FIELDS.howSatisfied, { required: 'Please select an option' }),
+        'inputRef',
+    );
+    const respondentNameProps = renameRefKey(register(FORM_FIELDS.respondentName), 'inputRef');
+    const respondentEmailProps = renameRefKey(register(FORM_FIELDS.respondentEmail), 'inputRef');
+
+    return (
+        <div id="feedback-form">
+            <h1>Give feedback on accessing Lloyd George digital patient records</h1>
+
+            <form onSubmit={handleSubmit(submit)}>
+                <Fieldset>
+                    <Fieldset.Legend size="m">What is your feedback?</Fieldset.Legend>
+                    <Textarea
+                        data-testid={FORM_FIELDS.feedbackContent}
+                        label="Tell us how we could improve this service or explain your experience using it. You
+                can also give feedback about a specific page or section in the service."
+                        rows={7}
+                        error={errors.feedbackContent?.message}
+                        {...feedbackContentProps}
+                    />
+                </Fieldset>
+
+                <Fieldset>
+                    <Fieldset.Legend size="m">
+                        How satisfied were you with your overall experience of using this service?
+                    </Fieldset.Legend>
+                    <Radios id="select-how-satisfied" error={errors.howSatisfied?.message}>
+                        {Object.values(SATISFACTION_CHOICES).map((choice) => (
+                            <Radios.Radio key={choice} value={choice} {...howSatisfiedProps}>
+                                {choice}
+                            </Radios.Radio>
+                        ))}
+                    </Radios>
+                </Fieldset>
+
+                <Fieldset>
+                    <Fieldset.Legend size="m">Leave your details (optional)</Fieldset.Legend>
+
+                    <p>
+                        If you’re happy to speak to us about your feedback so we can improve this
+                        service, please leave your details below.
+                    </p>
+
+                    <Input
+                        label="Your name"
+                        data-testid={FORM_FIELDS.respondentName}
+                        autoComplete="name"
+                        spellCheck={false}
+                        {...respondentNameProps}
+                    />
+
+                    <Input
```

Comment:
do we need an email type on this input or something else to validate it?

Reply:
Agree, adding email regex to prevent aws ses client error on submit with invalid email.

There is an example on the react-hook-form website

`<input
        id="email"
        {...register("email", {
          required: "required",
          pattern: {
            value: /\S+@\S+\.\S+/,
            message: "Entered value does not match email format",
          },
        })}
        type="email"
      />`

Reply:
Thanks both :D  Added email validation (using a validator library)
Now the email address can be either blank or a valid one. If an invalid email address is given, the form will show an alert upon submission.

Reply:
Is email a required field ?